manifest.yml

```diff
@@ -0,0 +1,4 @@
+---
+applications:
+- name: [PUT-PROTOTYPE-NAME-HERE]
```

Comment:
I think you are right. In the instructions we are now using `[NAME_OF_YOUR_APP]`. Better?

It is uppercase and in square parentheses to make it clearer which is the bit a user replaces. We've tried to use a similar style throughout the docs we've been testing, users haven't seemed to have issues knowing which bit to replace so far but happy for other suggestions.